internal/services/formatters/elixir/mixaudit/formatter.go

```diff
@@ -88,6 +88,7 @@ func (f *Formatter) parseOutput(output string) error {
 func (f *Formatter) newVulnerability(mixVuln *mixAuditVulnerability) *vulnerability.Vulnerability {
 	vuln := &vulnerability.Vulnerability{
 		SecurityTool: tools.MixAudit,
+		RuleID:       mixVuln.Advisory.CVE,
```

Comment:
I'm not familiar with this tool; is there a better way to synthesize a rule ID for issues raised here?

Reply:
Forget about the last comment. I was worried about printing this information into the final result. But in conversation with the team, we agreed with ignoring the rule ID for tools different from Horusec Engine during the print results. So, orchestrated rule IDs are going to be used only for the SARIF output.

Reply:
This ignore step should be implemented on this pr @nathanmartinszup ?

Reply:
Yes, otherwise we will print these unnecessary rules IDs.

This change can be made [here](https://github.com/ZupIT/horusec/blob/f14f9104d6851a2b705dd22a9363686e22509824/internal/controllers/printresults/print_results.go#L319).

I guess this is enough
```
	if vulnerability.RuleID != "" && vulnerability.SecurityTool == tools.HorusecEngine{
		pr.printlnf("RuleID: %s", vulnerability.RuleID)
	}
```

Reply:
@anthturner  

Also add a comment about why this validation is necessary, it can be confusing for other people why the other ID's are ignored.